Tests/Runner/RunnerTest.php

```diff
@@ -1851,4 +1851,45 @@ public function testExecutorStoreRowsUsage()
             ]
         ], $job->getUsage());
     }
+
+    /**
+     * @dataProvider swapFeatureProvider
+     */
+    public function testExecutorSwap($features)
+    {
+        $this->clearConfigurations();
+        $usageFile = new NullUsageFile();
+        $component = new Components($this->getClient());
+        $configuration = new Configuration();
+        $configuration->setComponentId('keboola.docker-demo-sync');
+        $configuration->setName('Test configuration');
+        $configuration->setConfigurationId('runner-configuration');
+        $component->addConfiguration($configuration);
+        $componentData = [
+            'id' => 'keboola.docker-demo-sync',
+            'data' => [
+                'definition' => [
+                    'type' => 'aws-ecr',
+                    'uri' => '147946154733.dkr.ecr.us-east-1.amazonaws.com/developer-portal-v2/keboola.python-transformation',
+                ],
+            ],
+            'features' => $features
+        ];
+        $configData = [
+            'parameters' => [
+                'script' => [],
+            ],
+        ];
+        $jobDefinition = new JobDefinition($configData, new Component($componentData), 'runner-configuration');
+        $runner = $this->getRunner();
+        $runner->run([$jobDefinition], 'run', 'run', '987654', $usageFile);
```

Comment:
ten test nema zadnej assert, idealni by bylo treba uvnitr spustit `free` a podivat se jestli je tam nejakej rozdil

Reply:
to mi přijde, že vůbec nic neřekne

```
[ondra@kbc-devel-03 docker-bundle]$ sudo docker run --rm --memory=256m --memory-swap=256m alpine:latest free
             total       used       free     shared    buffers     cached
Mem:       7663388    5956308    1707080          0    1088576    1322404
-/+ buffers/cache:    3545328    4118060
Swap:            0          0          0
[ondra@kbc-devel-03 docker-bundle]$ sudo docker run --rm --memory=256m --memory-swap=512m alpine:latest free
             total       used       free     shared    buffers     cached
Mem:       7663388    5959540    1703848          0    1088576    1322392
-/+ buffers/cache:    3548572    4114816
Swap:            0          0          0
[ondra@kbc-devel-03 docker-bundle]$ 
```

Reply:
assert jsem dávat nechtěl, chtěl jsem jen ověřit, že to v pořádku proběhne - takovej integrační test (setupovat to v KBC na kombinaci všech featur by byl fakt omrd)

Reply:
nj, tak kdyz nic nerekne, tak nerekne no :(

Reply:
chapu proc tam neni, ale nejakej assert by tam mel byt, jinak to na novym phpunitu nepojede

Reply:
doplněno. pustím na to branch testy na jenkinsi a pak to mergnu. dík